docs/guides/chip_tool_guide.md

```diff
@@ -1013,3 +1021,210 @@ In this command:
+    | * auto-relock-time                                                                  |
+    | * sound-volume                                                                      |
+    | * operating-mode                                                                    |
+    | * supported-operating-modes                                                         |
+    | * default-configuration-register                                                    |
+    | * enable-local-programming                                                          |
+    | * enable-one-touch-locking                                                          |
+    | * enable-inside-status-led                                                          |
+    | * enable-privacy-mode-button                                                        |
+    | * local-programming-features                                                        |
+    | * wrong-code-entry-limit                                                            |
+    | * user-code-temporary-disable-time                                                  |
+    | * send-pinover-the-air                                                              |
+    | * require-pinfor-remote-operation                                                   |
+    | * expiring-user-timeout                                                             |
+    | * generated-command-list                                                            |
+    | * accepted-command-list                                                             |
+    | * event-list                                                                        |
+    | * attribute-list                                                                    |
+    | * feature-map                                                                       |
+    | * cluster-revision                                                                  |
+    +-------------------------------------------------------------------------------------+
+    ```
+
+1. Add the argument of your choice to the subscription command, using the
+   following pattern:
+
+    ```bash
+    $ ./chiptool <cluster-name> subscribe <argument> <min-interval> <max-interval> <node_id> <endpoint_id>
+    ```
+
+    In this command:
+
+    - _<cluster-name\>_ is the name of the cluster.
+    - _<argument\>_ is the name of the chosen argument.
+    - _<min-interval\>_ specifies the minimum number of seconds that must elapse
+      since the last report for the server to send a new report.
+    - _<max-interval\>_ specifies the number of seconds that must elapse since
+      the last report for the server to send a new report.
+    - _<node-id\>_ is the user-defined ID of the commissioned node.
+    - _<endpoint_id\>_ is the ID of the endpoint where the chosen cluster is
+      implemented.
+
+    For example:
+
+    ```bash
+    $ ./chiptool doorlock subscribe lock-state 5 10 node_1 0
```

Comment:
```suggestion
    $ ./chiptool doorlock subscribe lock-state 5 10 1 1
```
Let use possible nodeid and endpoint e.g. fro our sample

Reply:
How about `$NODE_ID $ENDPOINT_ID`?  Then it can still be copy/pasted as long as you set the vars, but makes it clear that those are parameters you can change....  But either way works for me.

Also, this example should be running the command from inside interactive mode.